omnibus/config/projects/ddot.rb

```diff
@@ -33,7 +33,14 @@
   Omnibus::Config.git_cache_dir ENV["OMNIBUS_GIT_CACHE_DIR"]
 end
 
-INSTALL_DIR = ENV["INSTALL_DIR"] || '/opt/datadog-agent'
+if windows_target?
+  # Note: this is the path used by Omnibus to build the agent, the final install
+  # dir will be determined by the Windows installer. This path must not contain
+  # spaces because Omnibus doesn't quote the Git commands it launches.
+  INSTALL_DIR = 'C:/opt/datadog-agent/'
```

Comment:
trailing slash?

Reply:
🤷 Inherited from https://github.com/DataDog/datadog-agent/blob/34040aae4992aadb981702cc3d7db5ef14fad17f/omnibus/config/projects/agent.rb#L48

which has been there for many, many, years. But I agree that it doesn't make much sense, I can get rid of it.